lib/active_force/active_query.rb

```diff
@@ -33,12 +33,6 @@ def limit limit
       limit == 1 ? to_a.first : self
     end
 
-    def where args=nil, *rest
```

Comment:
I see. I moved the methods relating to the more complex `where` syntax up to `Query` so stuff like ` where('CreatedDate > ?', Date.today)` would work. Currently `Query` only accepts raw string conditions. You're right though, a lot of that code shouldn't be on `Query`.